views/components/Sidebar/SidebarFooter.jsx

```diff
@@ -10,51 +10,62 @@ import {
 } from '@mui/material'
 import { Link } from 'react-router-dom'
 import { routes } from '../../routes/routes'
-import { FontSize14InRem } from '../../../constants'
+import { fontSize } from '../../../constants'
 import './SidebarFooter.css'
 
 const SidebarFooter = ({ user, onLogout }) => {
   return (
-    <Card sx={{ border: 0, boxShadow: 0, color: 'text.primary' }}>
+    <Card sx={{ border: 0, boxShadow: 0, marginTop: 'auto' }}>
       <CardHeader
         avatar={<Avatar alt={user.display_name[0]} src={user.icon} />}
+        sx={{ pb: '8px' }}
       />
-      <CardContent sx={{ pt: 0 }}>
-        <Typography sx={{ fontWeight: 600 }} variant="h5">
-          {user.display_name}
-        </Typography>
+      <CardContent sx={{ pt: 0, pb: '8px' }}>
+        <Typography sx={{ fontWeight: 600 }}>{user.display_name}</Typography>
```

Comment:
why was the variant removed..?

Reply:
it was changing the font size and I didn't see the specific header value in the design so I removed it.